src/components/Review/DecisionArea.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import { Button, Header, Modal, Radio, Segment, TextArea, TextAreaProps } from 'semantic-ui-react'
+import { SummaryViewWrapper } from '../../formElementPlugins'
+import { ReviewResponseDecision } from '../../utils/generated/graphql'
+import { DecisionAreaState, ReviewQuestionDecision } from '../../utils/types'
+import strings from '../../utils/constants'
+
+interface DecisionAreaProps {
+  state: DecisionAreaState
+  setDecision: Function
+  submitHandler: (_: any) => void
+  problemMessage: string
+  setProblemMessage: Function
+}
+
+const DecisionArea: React.FC<DecisionAreaProps> = ({
+  state,
+  setDecision,
+  submitHandler,
+  problemMessage,
+  setProblemMessage,
+}) => {
+  const { open, review, summaryViewProps } = state
+  const handleChange = (value: ReviewResponseDecision) => {
+    const { id, comment } = review as ReviewQuestionDecision
+    setProblemMessage('')
+    setDecision({
+      ...state,
+      review: { id, comment, decision: value },
+    })
+  }
+
+  const handleUpdateComment = (_: any, { value }: TextAreaProps) => {
+    const { review } = state
+    setDecision({ ...state, review: { ...review, comment: value } })
+  }
+
+  return (
+    <Modal
```

Comment:
If it's easy to do -- clicking anywhere outside the modal should close it, don't you think?

Reply:
Not easy. But probably a nice one to add to the Epic# 10 #78.

Reply:
Okay no probs for now. It's odd though, the default Modals on the React Semantic site all have this behaviour by default. Any idea why?

Reply:
Yes. I can explain in person, since it's a long history : D